### PR TITLE
17 migrate web site to markdown and mkdocs

### PR DESCRIPTION
**Pull request by reece**
This PR closes out the automated migration phase of varnomen pages (see #17 for the migration plan). The goal of this phase is to migrate varnomen pages approximately as-is, with all content in markdown, links working, and automated builds (at https://hgvs-nomenclature.org/en/17-migrate-web-site-to-markdown-and-mkdocs/).

Manual fixes (e.g., typos and broken links), new or edited content, and stylistic changes will be undertaken next but are intentionally excluded from this phase.

Most links work. A spot check of 5 broken links all resulted from links that are broken in varnomen, and therefore will be fixed later. 

Update 1: I have subsequently discovered links that are broken due to migration. However, they are few enough that we can fix manually later. So, to be clear, some broken links are expected and I request that you ignore them at this stage.

### CODE REVIEW

**Review by ahwagner — APPROVED** [review]
@reece this is outstanding. The effort that you have put into programmatically identifying and resolving reference and formatting inconsistencies has dramatically improved the overall look and feel of the new site.

I spent a little time reviewing the [documentation build](https://hgvs-nomenclature.org/en/17-migrate-web-site-to-markdown-and-mkdocs/) as well–what a great way to showcase this system.

The only feature I can think to suggest as an enhancement would be to add in an RTD webhook for [automatic PR builds](https://docs.readthedocs.io/en/stable/pull-requests.html). This shouldn't affect the PR at all, since it would be a repo administration task.

Well done, 👍 .

File: bin/adapt-varnomen-pages [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    return content
+
+
+def process_hvnc(content: str) -> str:
+    content = re.sub(r"### (Call for members)\n\n(.+?)\n\n", r'!!! note "\1"\n\n    \2\n\n', content)
+    return content
+
+def process_recommendation(path) -> str:
+    def rewrite_discussion(d):
+        v = re.sub(r"^\s+([^:]+):\s*(.+)", r'\n!!! note "\1"\n\n    \2', d.group(1), flags=re.MULTILINE)
```
Comment (ifokkema) [inline bin/adapt-varnomen-pages:93]: The end of the question isn't always the first colon. The separator may be `?:`, but I didn't test if that applies to all pages. E.g., the last three comments [on this page](https://hgvs-nomenclature.org/en/17-migrate-web-site-to-markdown-and-mkdocs/recommendations/DNA/substitution/) are partially included in the body of the note. Splitting on `?:` and putting back the `?` at the end of the question would split the notes on that page correctly.

**Comment by reece** [comment]
Good catches, @ifokkema !

Discussion parsing was fixed (using `: `, with a space) in 7077e6e and page ordering was added in 2d1cc73. 

I also merged these changes into the manual-website-updates branch, so both should be current.

**Review by ifokkema — COMMENTED** [review]
Everything related to conversions can be removed, as conversions [have been replaced with deletion-insertions](https://varnomen.hgvs.org/bg-material/consultation/svd-wg009/). In principle, we could keep the sources (as is done now), and leave out the references, but we could also clean up and remove all mentions of conversions.

**Pull request merged**